### PR TITLE
add SYSTEM STOP/START FETCHES and SYNC REPLICA description

### PR DESCRIPTION
**Pull request by Slach**
Signed-off-by: Slach <bloodjazman@gmail.com>
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/sql-reference/statements/system.md [hunk 1]
```diff
@@ -108,4 +111,25 @@ Provides possibility to start background merges for tables in the MergeTree fami
...
 
+### STOP FETCHES {#query_language-system-stop-fetches}
+Provides possibility to stop background fetches for inserted parts for tables in the ReplicatedMergeTree family:
+
+``` sql
+SYSTEM STOP FETCHES [[db.]replicated_merge_tree_family_table_name]
+```
+
+### SYNC REPLICA {#query_language-system-sync-replica}
+Wait when ReplicatedMergeTree table will synced with other replicas in cluster, will run until receive_timeout if fetches currently disabled for table:
```
Comment (blinkov) [inline docs/en/sql-reference/statements/system.md:122]:
```suggestion
Wait until a `ReplicatedMergeTree` table will be synced with other replicas in a cluster. Will run until `receive_timeout` if fetches currently disabled for the table.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/sql-reference/statements/system.md [hunk 2]
```diff
@@ -108,4 +111,25 @@ Provides possibility to start background merges for tables in the MergeTree fami
 SYSTEM START MERGES [[db.]merge_tree_family_table_name]
 ```
 
+### STOP FETCHES {#query_language-system-stop-fetches}
+Provides possibility to stop background fetches for inserted parts for tables in the ReplicatedMergeTree family:
```
[the remark this answers is not in the document]
Comment (Slach) [reply docs/en/sql-reference/statements/system.md:115]: nothing happens when I ran `SYSTEM STOP FETCHES` as I see ;)

[the remark this answers is not in the document]
Comment (Slach) [reply docs/en/sql-reference/statements/system.md]: @blinkov could you provide any side effects when STOP FETCHES will run twice and more?

**Comment by Slach** [comment]
@blinkov could you look this PR again? I add all missed SYSTEM statements descriptions

**Comment by blinkov** [comment]
> @blinkov could you look this PR again? I add all missed SYSTEM statements descriptions
It looks like there's an issue with an incorrect link on single-page version:
```
INFO:root:Tag <a href="#query_language-system-restart-replicas">RESTART REPLICAS</a>
INFO:root:Link to nowhere: #query_language-system-restart-replicas
```

**Comment by Slach** [comment]
@blinkov link fixed on https://github.com/ClickHouse/ClickHouse/pull/11444/commits/62707935307688a6ff0f321674118d4ae6c8f31d

**Pull request merged**